webpack/assets/javascripts/react_app/components/CPUCoresInput/cpuCoresInput.scss

```diff
@@ -1,6 +1,12 @@
 
-.help-block-icon {
+.warning-icon {
   margin-right: 3px;
+  color: #f0ab00;
+}
+
+.error-icon {
+  margin-right: 3px;
+  color: #c9190b;
 }
```

Comment:
Can you use the colors from this file?
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/colors.scss